bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -78,6 +78,9 @@ public WebViewEnvironment(ICoreWebView2Environment environment) {
 	private boolean ignoreFocus;
 	private String lastCustomText;
 
+	private record CursorPosition(Point location, boolean isInsideBrowser) {};
```

Comment:
```suggestion
	private static record CursorPosition(Point location, boolean isInsideBrowser) {};
```

Reply:
Aren't records implicitly static?

Reply:
I think they are only implicitly static in specific situations, such as when defined locally (within a method). So I think adding the static modifier makes sense. I will update the PR accordingly. Thank you for the pointer!

Reply:
Fixed in 77a5a81